common.blocks/demo/demo.js

```diff
@@ -54,9 +69,25 @@ modules.define('demo', [ 'i-bem__dom', 'pretty', 'functions__debounce' ], functi
                 return;
             }
 
-            this._html.setValue(pretty(bemhtml.apply(BEMJSON), {
-                max_char: 1000
-            }));
+            var compiledText = '';
+
+            switch(this.params.engine) {
+                case 'vidom':{
+                    compiledText = JSON.stringify(bemhtml.apply(BEMJSON), null, 4);
+                    break;
+                }
+                case 'bemhtml':{
+                    compiledText = pretty(bemhtml.apply(BEMJSON), { max_char: 1000});
+                    break;
+                }
+                case 'bemjson':{
+                    compiledText = JSON.stringify(BEMJSON, null, 4);
```

Comment:
Can we make `Engine.render(BEMJSON)` method to incapsulate there logic about rendering for engines?